### PR TITLE
adds declaration-block-no-shorthand-property-overrides rule

### PR DESCRIPTION
**Pull request by rowbi-wan**
## Suggested rule/changes(s):
* `declaration-block-no-shorthand-property-overrides` : `severity: WARNING`

## Reason for addition/amendment
prevents shorthand properties from overriding longhand ones. 

@netsells/frontend - Please review

### CODE REVIEW

File: rules/core/declaration-block-no-shorthand-property-overrides /correct.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+a {
+    padding: 10px;
+    padding-left: 20px;
+}
+
+
```
Comment (samturrell) [inline rules/core/declaration-block-no-shorthand-property-overrides /correct.css]: Lot of spacing

**Review by samturrell — CHANGES_REQUESTED** [review]

File: rules/core/declaration-block-no-shorthand-property-overrides /correct.css (hunk 1 above)
Comment (rowbi-wan) [reply rules/core/declaration-block-no-shorthand-property-overrides /correct.css]: yeah was trying to show that the one in the middle go together as a single example

**Comment by samturrell** [comment]
Could always update the generator to load in `incorrect1.css` `incorrect2.css` etc :P

**Comment by rowbi-wan** [comment]
I was thinking an easier solution to this would be for me to just add a comment to the css file?

**Review by samturrell — APPROVED** [review]

**Pull request merged**